adaptor/elasticsearch/clients/v5/writer.go

```diff
@@ -85,6 +87,11 @@ func (w *Writer) Write(msg message.Msg) func(client.Session) (message.Msg, error
 			id = msg.ID()
 			msg.Data().Delete("_id")
 		}
+		var pID string
+		if _, ok := msg.Data()[w.parentID]; ok {
+			pID = msg.Data()[w.parentID].(string)
```

Comment:
will it ever be possible for the value to *not* be a `string`?

Reply:
It seems like Elasticsearch always stores _id as strings, this parentID should always be a string.

Can't seem to find it directly in the elasticsearch documentation, but the general consensus seems to be that _id automatically gets converted to string.

https://stackoverflow.com/questions/36597320/elasticsearch-document-id-type-integer-vs-string-is-there-any-performace-diffe

https://discuss.elastic.co/t/is-it-possible-to-use-string-as--id/17236/2

http://grokbase.com/t/gg/elasticsearch/151vqsb0vf/how-does-sorting-on-id-work

Reply:
👍